tests/bearlib/languages/documentation/documentation_extraction_testdata/default.py

```diff
@@ -37,3 +37,30 @@ def best_docstring(param1, param2):
              Cut to the Next Line.
     """
     return None
+
+def docstring_find(filename):
+    """
+    This is dummy docstring find function!
+
+    :param filename:
+        contains filename
```

Comment:
Ok so I was correct, only descriptions are Sentence case. Rest all desc. of params, exceptions, returns are lower case. Here check [PEP-257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings). Also same is being followed in [Java docstyle](http://www.oracle.com/technetwork/articles/java/index-137868.html). But still there are some inconsistencies in my implementation fixing them. I will create newcomers issue for the pre-existing examples :+1: